common/src/api/external/mod.rs

```diff
@@ -919,7 +944,7 @@ impl From<crate::api::internal::nexus::InstanceRuntimeState>
     }
 }
 
-/// Client view of an [`Instance`]
+/// View of an Instance
```

Comment:
Would this be a good place for us to (eventually) put the user-facing overview of each resource?  e.g., https://stripe.com/docs/api/products (though I was imagining a fair bit more detail).

GCP has this sort of thing at https://cloud.google.com/compute/docs/instances, though worth noting that that the REST resource itself is really more of a stub that _points_ to that guide: https://cloud.google.com/compute/docs/reference/rest/v1/instances

Reply:
We were discussing that earlier this week. It's something I'd like to have on the docs site.
We could do something with the API spec, with AsciiDoc, or both.

I'm sure I could grab the resource object from its schema ref, e.g. "#/components/schemas/Instance".

Reply:
Yes. That makes a ton of sense.